OQ_Toolkit/OQ_ARVROrigin/scripts/Locomotion_Stick.gd

```diff
@@ -65,8 +67,19 @@ func move(dt):
 		
 	if (enable_vignette) : movement_vignette_rect.visible = true;
 		
-	var view_dir = -vr.vrCamera.global_transform.basis.z;
-	var strafe_dir = vr.vrCamera.global_transform.basis.x;
+	var view_dir: Vector3
+	var strafe_dir: Vector3
+	
+	match movmenet_orientation:
+		MovementOrientation.HEAD:
+			view_dir = -vr.vrCamera.global_transform.basis.z;
+			strafe_dir = vr.vrCamera.global_transform.basis.x;
+		MovementOrientation.HAND_RIGHT:
+			view_dir = -vr.rightController.global_transform.basis.z;
```

Comment:
Just out of curiosity: is the hand-relative movement something common? I have not yet seen this used but I also don't play a lot of VR games. For what kind of games/applications is this an option that users would choose?

Reply:
Oh ya, it's in every VR game I can think of that I play. (_Half-Life: Alyx is the first one I can think of_) I personally always do Left hand relative (also the move stick on left, snap turn on right).